src/components/Line.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core';
+
+const styles = {
+  root: {
+    width: '80px',
+    height: '2px',
+    background: 'lightgrey',
+    marginTop: '20px',
+    borderLeft: 'solid #023256 20px'
```

Comment:
I think these are mobile screens. For desktop, they should be `75px/0.7px` and border of `22.5px` wide (and `1.5px` height)

Reply:
Thats not the text section 

Thats addressed here https://github.com/CodeForAfricaLabs/sensors.AFRICA-SeaSensors/pull/34

Reply:
Will address this on another style fix